src/utilities/findBreakingChanges.js

```diff
@@ -190,7 +190,7 @@ function findTypeChanges(
     } else if (isObjectType(oldType) && isObjectType(newType)) {
       schemaChanges.push(...findObjectTypeChanges(oldType, newType));
     } else if (isInterfaceType(oldType) && isInterfaceType(newType)) {
-      schemaChanges.push(...findFieldChanges(oldType, newType));
+      schemaChanges.push(...findInterfaceTypeChanges(oldType, newType));
```

Comment:
Don't think it's worth to have `findObjectTypeChanges` and `findInterfaceTypeChanges` as wrapper functions. Please change it to:
```
schemaChanges.push(
  ...findImplementedInterfacesChanges(oldType, newType),
  ...findFieldChanges(oldType, newType),
);
```
and do the same for object type.

Reply:
done